### PR TITLE
PT-640 Display of parameter history is not accurate

### PR DESCRIPTION
**Pull request by yvon-dblg**
Hot fix.

### CODE REVIEW

File: src/components/settings/DiabeloopHistoryParameters.js [hunk 1]
```diff
@@ -118,73 +118,88 @@ export default class HistoryTable extends Table {
...
+      const nHistory = history.length;
+      for (let i = 0; i < nHistory; i++) {
+        const parameters = history[i].parameters;
+
+        if (_.isArray(parameters)) {
+          const nParameters = parameters.length;
+          let latestDate = new Date(0);
+
+          // Compare b->a since there is a reverse order at the end
+          parameters.sort((a, b) => b.name.localeCompare(a.name));
```
Comment (franck-fourel) [inline src/components/settings/DiabeloopHistoryParameters.js]: As we have now different levels of parameters, I just realized we should sort on 2 fields: level and name. Can you apply the below change:
```suggestion
          parameters.sort((a, b) =>  
            b.level.toString().localeCompare(a.level.toString())
            || b.name.localeCompare(a.name)
            );
```

**Review by franck-fourel — CHANGES_REQUESTED** [review]
little change for sorting

**Review by franck-fourel — APPROVED** [review]

**Comment by franck-fourel** [comment]
@yvon-dblg can you please review the change I did on the sorting in 4deb40706a83afd6063634a2969ebdfc9f290a37

**Review by yvon-dblg — COMMENTED** [review]
OK LGTM

**Review by ydrisR — APPROVED** [review]
LGTM

**Pull request merged**